doc/install_mne_c.rst

```diff
@@ -2,290 +2,175 @@
 
 .. _install_mne_c:
 
-Install MNE-C
--------------
+Installing MNE-C
+================
 
-The MNE-C commands can be downloaded
-`here <http://www.nmr.mgh.harvard.edu/martinos/userInfo/data/MNE_register/index.php>`_.
-The :ref:`c_reference` gives an overview of the MNE-C tools.
-
-.. contents:: Contents
+.. contents::
    :local:
    :depth: 1
 
-System requirements and installation
-####################################
-
-The MNE-C runs on Mac OSX and LINUX, and requires:
-
-- Mac OSX version 10.5 (Leopard) or later.
-- LINUX kernel 2.6.9 or later
-- On both LINUX and Mac OSX 32-bit and 64-bit Intel platforms
-  are supported. PowerPC version on Mac OSX can be provided upon request.
-- At least 2 GB of memory, 4 GB or more recommended.
-- Disk space required for the MNE software: 80 MB
-- Additional open source software on Mac OSX, see :ref:`BABDBCJE`.
-
-The MNE software is distributed as a compressed tar archive
-(Mac OSX and LINUX) or a Mac OSX disk image (dmg).
+System requirements
+^^^^^^^^^^^^^^^^^^^
 
-The file names follow the convention:
+MNE-C runs on macOS (version 10.5 "Leopard" or later) and Linux (kernel 2.6.9
+or later). Both 32- and 64-bit operating systems are supported; a PowerPC
+version for macOS can be provided upon request. At least 2 GB of memory is
+required, 4 GB or more is recommended. The software requires at least 80 MB of
+disk space. MATLAB is an optional dependency; the free `MATLAB runtime`_ is
+sufficient. If MATLAB is not present, the utilities ``mne_convert_mne_data``,
+``mne_epochs2mat``, ``mne_raw2mat``, and ``mne_simu`` will not work.
 
-::
+For boundary-element model (BEM) mesh generation (see :doc:`Creating the BEM
+meshes <manual/appendix/bem_model>`), and for accessing the ``tkmedit`` program
+from ``mne_analyze`` (see :ref:`CACCHCBF`), MNE-C needs access to a working
+installation of :doc:`FreeSurfer <install_freesurfer>`, including the
+environment variables ``FREESURFER_HOME``, ``SUBJECTS_DIR``, and ``SUBJECT``.
 
-  MNE-* <*version*>*- <*rev*> -* <*Operating system*>*-* <*Processor*>*.* <*ext*>*
+.. admonition:: |apple| macOS
+  :class: note
 
-The present version number is 2.7.4. The <*rev*> field
-is the SVN revision number at the time this package was created.
-The <*Operating system*> field
-is either Linux or MacOSX. The <*processor*> field
-is either i386 or x86_64. The <*ext*> field
-is 'gz' for compressed tar archive files and 'dmg' for
-Mac OSX disk images.
+  For installation on macOS, you also need:
 
-Installing from a compressed tar archive
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+  - the `XCode developer tools`_.
+  - an X Window System such as XQuartz_. Specifically, you need version 2.7.9
+    of XQuartz; the most current version (2.7.11, as of Feb. 2019) will not
+    work.
+  - the netpbm_ library. The recommended way to get netpbm is to install
+    MacPorts_, and then run ``sudo port install netpbm`` in the Terminal app.
```

Comment:
macports is pretty dead now. Is this still up to date? needed? Would brew do the job too?

Reply:
I didn't know macports was dead... they had a release 4 months ago and their github repo's last change was 29 days ago...  In any case, I don't have any insight into what's best nowadays since I don't have access to a mac computer.

There is [a brew formula for netpbm](https://brewformulas.org/Netpbm), so if someone else can confirm that it works (or if you sufficiently trust that `brew install netpbm` will work) then I can change the instructions, no problem.

Reply:
I confirm that I use

brew install netpbm

and that it works for me. Just add that `brew install netpbm` is an alternative and for which sudo should not be needed. Thanks

Reply:
fixed in b447cc1